docs/WindowsBuild.md

```diff
@@ -103,6 +93,10 @@ Extract the zip files, ignoring the top level directory, into `S:/Library`. The
       ┕ usr/...
 ```
 
+Note that only ICU is required for building the toolchain, and SQLite is only
+needed for building llbuild and onwards.  The ICU project provides binaries,
+alternatively, see the ICU project for details on building ICU from source.
```

Comment:
Added article `the`

```suggestion
alternatively, see the ICU project for details on building ICU from the source.
```

Reply:
AFAIK, the expression is "from source" when referencing source code, which is why I think that the original statement is correct.  Is there a definite guide to whether it should be "from the source" or "from source"?  The former seems to imply that there is the singular source, where as it may be a fork or from a mirror.